shared/src/business/entities/cases/Case.ts

```diff
@@ -651,6 +655,18 @@ export class Case extends JoiValidationEntity {
       .required()
       .description('Procedure type of the case.')
       .messages({ '*': 'Select a case procedure' }),
+    remoteTrialGranted: joi
+      .boolean()
+      .optional()
+      .description(
+        'Whether the case has been granted a motion to proceed remotely.',
+      )
+      .meta({ tags: ['Restricted'] }),
+    remoteTrialGrantedDate: JoiValidationConstants.ISO_DATE.optional()
+      .allow(null)
+      .description(
+        'A date stamp indicating the date permission was granted to proceed remotely.',
+      ),
```

Comment:
Not a big deal, but we could prevent a few edge cases by making this optional when `remoteTrialGranted` is undefined, but required when it is defined.